apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -3398,11 +3425,16 @@ private static Map<String, String> readNpiCodeFile() {
         final BufferedReader npiCodesIn =
             new BufferedReader(new InputStreamReader(npiCodeDisplayStream))) {
       /*
-       * We want to extract the NPI codes and display values and put in a map for easy retrieval to
-       * get the display value-- npiColumns[0] is the NPI Code, npiColumns[4] is the NPI
-       * Organization Code, npiColumns[8] is the NPI provider name prefix, npiColumns[6] is the NPI
-       * provider first name, npiColumns[7] is the NPI provider middle name, npiColumns[5] is the
-       * NPI provider last name, npiColumns[9] is the NPI provider suffix name, npiColumns[10] is
+       * We want to extract the NPI codes and display values and put in a map for easy
+       * retrieval to
```

Comment:
yea, the formatter can be irksome....my worst example when saving a file:

HospiceClaim hospiceClaim =
            parsedRecords.stream()
                .filter(r -> r instanceof HospiceClaim)
                .map(r -> (HospiceClaim) r)
                .findFirst()
                .get();
                
No matter what, the formatter would remove the cast from lambda.

Reply:
Saving a file? So it is your IDE? I imagine there are settings for that.

Reply:
What's interesting is that when using multiple single line `//` comments the Google formatter will rewrap a paragraph just fine.  I suggest using multiple `//` instead of `/*` as a way of avoiding this issue.  

Also, using the google format plugin in IDEA makes it easy to see what will happen when the formatter maven plugin runs.